src/sentry_plugins/bitbucket/plugin.py

```diff
@@ -205,3 +238,137 @@ def view_autocomplete(self, request, group, **kwargs):
         } for i in response.get('issues', [])]
 
         return Response({field: issues})
+
+    def get_configure_plugin_fields(self, request, project, **kwargs):
+        return [{
+            'name': 'repo',
+            'label': 'Repository Name',
+            'default': self.get_option('repo', project),
```

Comment:
iirc this default thing isnt required anymore, but you'd want to test to confirm